test/test_eos.py

```diff
@@ -54,8 +55,17 @@
 from pyopencl.tools import (  # noqa
     pytest_generate_tests_for_pyopencl as pytest_generate_tests,
 )
+if sys.version_info < (3, 9):
+    # importlib.resources either doesn't exist or lacks the files()
+    # function, so use the PyPI version:
+    import importlib_resources
+else:
+    # importlib.resources has files(), so use that:
+    import importlib.resources as importlib_resources
+
 
 logger = logging.getLogger(__name__)
+mechdata = importlib_resources.files("mechanisms")
```

Comment:
We could package this up in a function, so we don't have to repeat it? `get_mechanism_path`?

Reply:
904b6bc  Yeah, I think this is much better.